lua/ttt2/libraries/entspawnscript.lua

```diff
@@ -514,9 +520,7 @@ function entspawnscript.RemoveSpawnById(spawnType, entType, id)
 
 	tableRemove(list, id)
 
-	if SERVER then
-		entspawnscript.UpdateDataOnClients()
-	else -- CLIENT
+	if CLIENT then
```

Comment:
these following three changes seem to break with my most recent addition. However your branch doesn't seem to be up to date as well. So it might be resolved by updating your branch.